Makefile

```diff
@@ -130,14 +140,14 @@ install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~
 	$(KUSTOMIZE) build config/crd | kubectl apply -f -
 
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | kubectl delete -f -
+	$(KUSTOMIZE) build config/crd | kubectl delete --ignore-not-found=true -f -
 
```

Comment:
Why adding `--ignore-not-found=true` is needed?

Reply:
I saw a failure while performing make install, make deploy, make undeploy and make uninstall so added this `--ignore-not-found=true`